apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/BFD-2572-Cleanup-Medicare-Beneficairyid-History/invoke-psql.sh

```diff
@@ -0,0 +1,345 @@
+#!/bin/bash
+# ------------------------------------------------------------------
+# Shell script to cleanup spurious rcds in the MEDICARE_BENEFICIARYID_HISTORY
+# table.
+#
+# The MEDICARE_BENEFICIARYID_HISTORY is a BFD dtabase table that (mostly) represents
+# data provided as a one-time data drop by CCW (MAR-2018).
```

Comment:
The PR says `APR-2019`, is it actually `MAR-2018`?

Reply:
PR is correct...I wrote up the script prior to x-checking when that CCW data dump was loaded. It was APR-2019.